charts/jaeger/values.yaml

```diff
@@ -613,14 +639,6 @@ query:
     pathType:
     health:
       exposed: false
-  resources:
```

Comment:
you mean for query, ingress has no resources. I think this was a mistake and almost twice. It was a bit confusing to walk all the values over the time. If it would make sense to have this in alphabetically order? Then I would do this on another PR

Reply:
Sorry for the late response on this, I'd prefer refactoring in another PR.